### ISSUE TITLE
https://quarkus.io/guides/application-lifecycle-events-guide issues for  0.13.1

### ISSUE THREAD

**Issue by paulvi** [quarkusio/quarkusio.github.io#173]
I am trying out https://quarkus.io/guides/application-lifecycle-events-guide using 0.13.1 quarkus

and it just fails to run

```
D:\Workspaces\eclipse-jee-2019-03\quarkusmy\application-lifecycle-events>mvn compile quarkus:dev
[INFO] Scanning for projects...
[INFO]
[INFO] ---------------< org.acme:application-lifecycle-events >----------------
[INFO] Building application-lifecycle-events 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ application-lifecycle-events ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ application-lifecycle-events ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to D:\Workspaces\eclipse-jee-2019-03\quarkusmy\application-lifecycle-events\target\classes
[INFO]
[INFO] --- quarkus-maven-plugin:0.13.1:dev (default-cli) @ application-lifecycle-events ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.021 s
[INFO] Finished at: 2019-04-12T16:45:12+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:0.13.1:dev (default-cli) on project application-lifecycle-events: Failed to run: Failed to resolve Quarkus application model: Failed to locate current project among the loaded local projects -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

```

> Failed to run: Failed to resolve Quarkus application model: Failed to locate current project among the loaded local projects

**Comment by paulvi** [comment]
The app from https://github.com/quarkusio/quarkus-quickstarts/tree/master/application-lifecycle-events can be run OK, but not as within current guide text version

**Comment by gsmet** [comment]
@paulvi I think the issue might be fixed in 0.14.0.

**Comment by gsmet** [comment]
Closing as outdated.
